examples/workdir/env/extravars

```diff
@@ -4,6 +4,17 @@ ceph_user_keyring_b64key: QVFDZWoySmNBQUFBQUJBQWlhSTZJZEw2Nml4S1FINEVlOTBTbXc9PQ
 # This is a list of the Ceph MONs, should the ses_config.yml to override the
 # value provided by the ses_config.yml (in case of dual stack, floating IPs...)
 suse_osh_deploy_ceph_mons: ['192.168.50.10:6789']
+
 # This is the VIP that will be used for the openstack deployment on kubernetes
 # (ingress controller)
-suse_osh_deploy_vip_with_cidr: 192.168.50.46/24
```

Comment:
If this is removed, then please update the code to not break the deploy-osh, as it seems not updated.